locale/en_US/submission.xml

```diff
@@ -547,6 +550,11 @@
 	<message key="editor.submission.decision.approveProofs">Approve Proofs</message>
 	<message key="editor.submission.decision.disapproveProofsDescription">This file proof will no longer be publicly available for download or purchase. Do you want to disapprove it?</message>
 	<message key="editor.submission.decision.noDecisionsAvailable">Assign an editor to enable the editorial decisions for this stage.</message>
+	<!--  editor recommendations -->
+	<message key="editor.submission.recommend">Recommend</message>
+	<message key="editor.submission.recommendation">Recommendation</message>
+	<message key="editor.submission.recommendation.description">Make a recommendation for the editors</message>
```

Comment:
Recommend an editorial decision be made for this submission.

Reply:
Do you maybe mean: Recommend an editorial decision for this submission.

Reply:
Yeah that's better. :)